schemas/hub/manifest.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$id": "https://raw.githubusercontent.com/guardrails-ai/interfaces/schemas/hub/manifest.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Manifest",
+  "type": "object",
+  "properties": {
```

Comment:
seems like a lot of properties in one place. What do you think of (later) splitting this up into the following categories?
1. github properties
2. hub-entry properties (submitter/id)
3. runtime properties

Reply:
yeah namespacing